Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.cpp

```diff
@@ -395,7 +395,12 @@ void RemoteRenderingBackendProxy::didCreateImageBufferBackend(ImageBufferBackend
     auto imageBuffer = m_remoteResourceCacheProxy.cachedImageBuffer(renderingResourceIdentifier);
     if (!imageBuffer)
         return;
-
+    
+    // If web process tried to create an accelerated buffer but gets that it created an unaccelerated buffer from the GPU process
+    // fallback to unaccelerated buffer in the web process
```

Comment:
I think this comment needs word smithing and I am not sure if it adds much value than describing the code below it.

Reply:
I removed this comment.